### DISCUSSION TITLE
Using SMPTE (ruler as time instead of measure/beats)

### DISCUSSION THREAD

**Discussion by daeavelwyn** [helgoboss/helgobox#1377] [Q&A]
Hello,

For live performance with dance / movie sync, I need to use SMPTE time (hh:mm:ss:frame) instead of time signature, measure and beats. Is it possible to acheive this with playtime ?

**Comment by helgoboss** [comment]
I think you are asking for two things: Sync and ruler format. Both are supported in REAPER, as described below.

###  Syncing to external SMPTE timecode

1. Right-click REAPER's play button
2. Configure SMPTE syncing

<img width="577" alt="image" src="https://github.com/user-attachments/assets/e341c850-4778-4cb4-a300-e7fe86d8897e" />

Whenever REAPER is playing, Playtime plays in sync to it. Which means, this feature should work with Playtime as well.

You are welcome to try. Both products have trial versions.

### Displaying the current time and ruler in hh:mm:ss:frame format.

1. Right-click the ruler
2. Choose "Hours:Minutes:Seconds:Frames"

**Comment by daeavelwyn** [comment]
Hello,

Thanks for your reply, it helps a lot.
